Tests/scripts/infrastructure_tests/common_test.py

```diff
@@ -1,147 +1,262 @@
-from Tests.scripts import common
+from pathlib import Path
+from Tests.scripts.common import get_reviewer, get_person_in_charge, are_pipelines_in_order, is_pivot, get_slack_user_name
+from requests_mock import MockerCore
 
 
-def test_person_in_charge(mocker):
+def test_get_person_in_charge(mocker):
     """
-    Given a commit object
-    When person_in_charge is called on it
-    Then it should return the expected name, email and PR link
+    Given:
+        A commit with author name and title
+    When:
+        The function get_person_in_charge is called with the commit
+    Then:
+        It should return the author name and the pull request URL
     """
     commit = mocker.Mock()
     commit.author_name = 'John Doe'
-    commit.author_email = 'john@doe.com'
     commit.title = 'Merge branch \'master\' into branch-name (#123)'
 
-    expected = ('John Doe', 'john@doe.com', 'https://github.com/demisto/content/pull/123')
+    result = get_person_in_charge(commit)
 
-    result = common.person_in_charge(commit)
-
-    assert result == expected
+    assert result == ('John Doe', 'https://github.com/demisto/content/pull/123')
 
 
-def test_are_pipelines_in_order_as_commits_true(mocker):
+def test_pipelines_are_in_correct_order__false(mocker):
+    """
+    Given:
+        Two pipelines with different creation dates
+    When:
+        The function are_pipelines_in_order is called with the two pipelines
+    Then:
+        It should return False if the pipelines are not in order based on their creation dates
     """
```

Comment:
```suggestion
    Given:
        Two pipelines that are out of order in respect to their creation time
    When:
        The function are_pipelines_in_order is called with the two pipelines
    Then:
        It should return False as the pipelines are out of order
    """
```
the Given-When-Then isn't good enough in the other UT as well.
hope my change in the text explains what i'm doing differently, if not, approach me.